DEVELOP.md

```diff
@@ -73,6 +73,7 @@ gulp                    # Compile the JS and start the automatic
 To regenerate the `bundled.js` file without using the live reloader, run:
 
 ```
+gulp peg
```

Comment:
No worries -- I find these discussion really helpful. I also think it makes
sense to to add `peg` into the `default` tag. I will update the PR
accordingly.

On Tue, Jun 9, 2015 at 1:13 PM Isaac Hodes <notifications@github.com> wrote:

> In DEVELOP.md
> <https://github.com/hammerlab/cycledash/pull/659#discussion_r32037989>:
>
> > @@ -73,6 +73,7 @@ gulp                    # Compile the JS and start the automatic
> >  To regenerate the `bundled.js` file without using the live reloader, run:
> >
> >  ```
> > +gulp peg
>
>  My general preference is to just remember one command that always
> results in a correct state
>
> That's what I'm suggesting with sticking peg into the default task; we're
> on the same page here :)
>
> @armish <https://github.com/armish> Sorry that such a small change is
> getting outsized discussion, but I agree that sticking the peg task into
> the default task list is the best way to handle this. Then we don't need
> to mention it in the README/DEVELOP.md at all.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/cycledash/pull/659/files#r32037989>.
>